uenv/wannier90/develop/daint/environtments.yaml

```
@@ -0,0 +1,16 @@
+gcc-env:
+  compiler:
+      - toolchain: gcc
+        spec: gcc
+  mpi:
+      spec: cray-mpich@8.1.29
+  unify: true
+  specs:
+  - cmake
+  - openblas threads=openmp
+  - wannier90+mpi 
```

You can (and should) specify the version
```suggestion
  - wannier90@develop+mpi
```